### ISSUE TITLE
`openSeaChest_NVMe --getFeatures 1` not getting the right Arbitration Burst value

### ISSUE THREAD

**Issue by jaythegreatman** [Seagate/opensea-operations#17]
### Problem

Arbitration Burst returned by `--getFeatures 1`  does not match the raw value returned by `--getFeatures list`

### Expected behavior

`--getFeatures 1`  gets the lower 3 bits of the raw value as Arbitration Burst

### How to reproduce

`openSeaChest_NVMe --getFeatures list` returns
```
 Feature ID     Raw Value (DWORD 0)
===============================
    01h         0x00000004
...
```

While `openSeaChest_NVMe --getFeatures 1` returns
```
        Arbitration & Command Processing Feature
=============================================
Hi  Priority Weight (HPW) :             0x00
Med Priority Weight (MPW) :             0x00
Low Priority Weight (LPW) :             0x00
Arbitration Burst    (AB) :             0x00
```

### Deployment information

Any

### Additional information

https://github.com/Seagate/opensea-operations/blob/63e829f0fda8cb629dd6dcafeab28025910f3e91/src/nvme_operations.c#L128
Here it's getting the lower 2 bits.
Please refer to _NVM Express Base Specification, Revision 2.0d, Page 312, Figure 318, Bits 02:00_

**Comment by vonericsen** [comment]
@jaythegreatman,

Thank you for reporting this bug. I will correct this today.

**Comment by vonericsen** [comment]
I am marking this closed since this fix was part of the [openSeaChest v24.08.1 release](https://github.com/Seagate/openSeaChest/releases/tag/v24.08.1).

Please feel free to reopen this if there are other issues with this output
